assets/dev-env.lando.template.yml.ejs

```diff
@@ -175,19 +175,19 @@ services:
     type: compose
     services:
       image: ghcr.io/automattic/vip-container-images/wordpress:<%= wordpress.tag %>
-#      command: /usr/local/bin/entrypoint.sh
-      command: sh -c "rsync -a --chown=www-data:www-data /wp/ /shared/; sleep infinity"
       volumes:
         - ./wordpress:/shared
         - type: volume
           source: scripts
           target: /scripts
           volume:
             nocopy: true
-#      environment:
-#        LANDO_NO_SCRIPTS: 1
-#        LANDO_NEEDS_EXEC: 1
-#    initOnly: true
+      environment:
+        LANDO_NO_USER_PERMS: 1
+        LANDO_NO_SCRIPTS: 1
+        LANDO_NEEDS_EXEC: 1
+    initOnly: true
+    entrypoint: /usr/bin/rsync -a --chown=${LANDO_HOST_USER_ID}:${LANDO_HOST_GROUP_ID} /wp/ /shared/
```

Comment:
`${LANDO_HOST_USER_ID}` and `${LANDO_HOST_GROUP_ID}` will be taken directly from the environment; we don't need to spawn a shell to pass the values to the program.

Because the container is init-only, we don't need Lando scripts to manage permissions either; `rsync` perfectly understands numeric IDs and does not require the presence of `/etc/passwd`. This allows us to optimize WordPress images.